src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.scss';
+import {
+  AddToDoList,
+  setTasks,
+  showTasks,
+  GetToDoList,
+  updateLocalStorage,
+} from './modules/AddToDoItems.js';
+import { clearAllCompletedTasks } from './modules/Completed.js';
+
+const newTodolistInput = document.querySelector('#add-to-do > input');
```

Comment:
The design looks great 👍  But as the previous reviewer mentioned, currently, you have an issue with selecting elements. The best way to refactor it is to add an id attribute to input elements on HTML and select it with querySelector. Also there is no element with an ID of add-to-do in your HTML file. After you make this refactoring, the project looks like the below. Kindly fix the design also make sure the delete and edit functionality work as intended. 
<img width="1440" alt="Screenshot 2023-03-04 at 4 30 10 PM" src="https://user-images.githubusercontent.com/25887253/222905185-936083b3-3036-468d-8d7a-a7073e48d2d3.png">

Reply:
- [ ] [REQUIRED] As stated by the previous reviewers, there is an issue with element selections.  It looks like the querySelector is not selecting the correct node. Kindly add a correct id/class attribute to the input elements on HTML so it can be properly selected with querySelector. 
- After this has been fixed, please make sure the delete and edit functionality work as stated by the project requirement.
<img width="987" alt="Screenshot 2023-03-04 at 15 15 32" src="https://user-images.githubusercontent.com/58692590/222907724-2930cfce-a43c-47b4-bfa2-ea5f4270a1d1.png">